test/addons/openssl-client-cert-engine/binding.gyp

```diff
@@ -0,0 +1,11 @@
+{
+  'targets': [
+    {
+      'target_name': 'testengine',
+      'type': 'shared_library',
+      'product_extension': 'engine',
+      'sources': [ 'testengine.cc' ],
+      'include_dirs': [ '../../../deps/openssl/openssl/include' ]
```

Comment:
@jbergstroem is referring to `./configure --shared-openssl` builds.  I'm fine with leaving it like this, we can address it if/when we add testing for that to the CI matrix.

Apropos the Windows build, no, I'm still working on that.